src/main/java/edu/harvard/iq/dataverse/engine/command/impl/DeleteCollectionQuotaCommand.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package edu.harvard.iq.dataverse.engine.command.impl;
+
+import edu.harvard.iq.dataverse.Dataverse;
+import edu.harvard.iq.dataverse.authorization.Permission;
+import edu.harvard.iq.dataverse.engine.command.AbstractVoidCommand;
+import edu.harvard.iq.dataverse.engine.command.CommandContext;
+import edu.harvard.iq.dataverse.engine.command.DataverseRequest;
+import edu.harvard.iq.dataverse.engine.command.RequiredPermissions;
+import edu.harvard.iq.dataverse.engine.command.exception.CommandException;
+import edu.harvard.iq.dataverse.engine.command.exception.IllegalCommandException;
+import edu.harvard.iq.dataverse.storageuse.StorageQuota;
+import java.util.logging.Logger;
+
+/**
+ *
+ * @author landreev
+ */
+@RequiredPermissions(Permission.ManageDataversePermissions)
```

Comment:
wondering about permissions, here and generally - i.e. these should be superuser, no? otherwise anyone with a collection could just change or remove their quota?

Reply:
On the previous question - not entirely redundant, no; in part because in the context of uploading through the UI there may be some appreciable time between uploading and deciding to click 'save', yes. Performing the check during the final save only doesn't seem perfect either, since it just feels wrong to allow the user to complete that UI upload, when they are clearly going over. 

On the question of the permissions, yes, of course, that's clearly wrong - should be readable to the owners, but setting/unsetting should be admin only... just haven't gotten to fixing - will do shortly.